src/translations/pl.json

```
@@ -0,0 +1,1152 @@
+    "selectUser": "Wybierz użytkownika",
+    "status": "{{active}} z {{total}} ról",
+    "tap_to_manage": "Dotknij, aby zarządzać rolami",
+    "unassigned": "Nieprzypisane"
+  },
+  "settings": {
+    "about": "O aplikacji",
+    "account": "Konto",
+    "active_unit": "Aktywna jednostka",
+    "app_info": "Informacje o aplikacji",
+    "app_name": "Nazwa aplikacji",
+    "arabic": "Arabski",
```

Biome (2.4.11)

[error] The key arabic was already declared.

(lint/suspicious/noDuplicateObjectKeys)